src/views/network/CreateNetwork.vue

```diff
@@ -95,8 +95,11 @@ export default {
     isAdvancedZoneWithoutSGAvailable () {
       for (const i in this.actionZones) {
         const zone = this.actionZones[i]
-        if (zone.networktype === 'Advanced' && zone.securitygroupsenabled !== true) {
-          return true
+        // Dont display isolated networks for advanced with sg enabled
+        if (this.resource.zoneid === zone.id &&
+          zone.networktype === 'Advanced' &&
+          zone.securitygroupsenabled === true) {
+          return false
```

Comment:
Guessing this might still break since this.resource is passed from autogenview which is generally the first item in the list. Can leave it as before and just change the final return to false

Reply:
@davidjumani  Lets say I change the code to
```
for (const i in this.actionZones) {
        const zone = this.actionZones[i]
        if (zone.networktype === 'Advanced' && zone.securitygroupsenabled !== true) {
          return true
        }
      }
      return false
```


If we have two zones, one without sg and one with sg, it will return for the first check inside the for loop right even for the zone with sg enabled?

Reply:
Yes, so in the form the user will select the zone in which to create the network, and probably need to add a check to exclude zones without SG enabled in the create isolated network form.
The above check hides it if all the zones have sg enabled or there are no zones present

Reply:
Yeah that works fine. I have made the code changes and tested it in setup with two zones. one with sg disabled and 1 with sg enabled . It doesnt show "zone name" with sg enabled for isolated networks. I have added screenshots